src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/CaseTagDto.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.digital.ho.hocs.casework.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseDataTag;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+public class CaseTagDto {
```

Comment:
There is a question about whether we need this - but I feel that this is more of a RESTful way of handling this POST. We can then use the createdAt for the business event thats emitted.